bitcoin/src/network/message.rs

```diff
@@ -17,14 +17,13 @@ use io::Read as _;
 use crate::blockdata::block;
 use crate::blockdata::transaction;
 use crate::network::address::{Address, AddrV2Message};
-use crate::network::{message_network, message_bloom};
+use crate::network::{message_network, message_bloom, message_merkleblock};
```

Comment:
Why is the module called `message_merkleblock` instead of just `merkleblock`?

Reply:
To fit in with the names of all the others. FTR I think they all need renaming though.

Reply:
I could rename them all as an initial patch if that is desired?